get-testnet-tokens.md

```diff
@@ -88,7 +108,7 @@ There are two ways of getting tokens. The faucets operated by the SDK team will
 - sdk-testnet - https://faucet.aepps.com/
 - sdk-edgenet - https://edge-faucet.aepps.com/
 
-In the next steps we're going to show you how to get tokens from the faucet for **sdk-edgenet**. The same proces could be followed for the **sdk-testnet**.
+In the next steps we're going to show you how to get tokens from the faucet for **sdk-testnet**. The same proces could be followed for the **sdk-edgenet**.
```

Comment:
should we still need this `The same proces could be followed for the **sdk-edgenet**.`

Reply:
@alekseytsekov is edgenet still alive?